docs/reference/storage-uri.md

```diff
@@ -69,15 +69,26 @@ The region or custom endpoint will be detected using the first successful method
 - `AWS_REGION` environment variable
 - Amazon’s instance metadata API [https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html)
 
-### S3-compatible Objective storage like Minio
+### S3-compatible Object Storage like Minio, Google Cloud Storage, and more.
 
 
-Quickwit can target other S3-compatible storage like MinIO.
+Quickwit can target other S3-compatible storage.
 This is done by setting an endpoint url in the `QW_S3_ENDPOINT` environment variable.
 
 In this case, the region will be ignored.
 
-For instance:
+Set up the endpoint url:
```

Comment:
We are using `URI` everywhere in the code base, it would be better to use it here

Reply:
Put "Example" and "Example for Google Cloud Storage" instead of a "Note" after, looks very clean